change/@azure-msal-common-23d68d34-05ea-4e90-9a2c-8e1b2774e6b2.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added functionality to MSAL-Node's HttpClient that supports custom http(s) agents",
```

To make this a little less verbose:
```suggestion
  "comment": "Add support for custom http(s) agents #5472",
```